app/models/content/concerns/has_children.rb

```diff
@@ -20,17 +20,19 @@ def present?
     true
   end
 
-  def reflow_contents
-    reflow_tree tree, ordinals
+  def reflow_contents(child)
+    @child = child
```

Comment:
since reflow_tree is a recursive method (see line#33) it felt weird to be re-passing in child into reflow_tree